front/components/navigation/config.ts

```diff
@@ -130,7 +130,8 @@ export const getTopNavigationTabs = (owner: WorkspaceType) => {
       href: `/w/${owner.sId}/builder/assistants`,
       isCurrent: (currentRoute: string) =>
         currentRoute.startsWith("/w/[wId]/builder/") ||
-        currentRoute === "/w/[wId]/a",
+        (!owner.flags.includes("data_vaults_feature") &&
+          currentRoute.includes("/apps/")),
```

Comment:
is there no risk to have another route with /apps/ , like in a folder named apps ?

Reply:
fixed